frontend-build-pipeline.yml

```diff
@@ -16,30 +29,30 @@ steps:
 
 - script: |
     npm ci
-    npx ng build --prod
+    npx ng build
   displayName: 'npm Build'
 
 # skipped failing unit tests
 # https://jasmine.github.io/tutorials/your_first_suite#section-Pending_Specs
 - script: |
-    npx ng test --watch=false
+    npx ng test --watch=false --browsers=ChromeHeadless
```

Comment:
we want to be collecting code coverage eventually, doesn't have to be this PR but it should be soon

ng lint is also a good thing to include in an angular pipeline.

Reply:
Implemented eslint and setup some default rules that all give warnings instead of errors to reduce friction with the devs. I let them know that once they get all the warnings cleared out they can switch those to errors if they want the pipeline to halt on linting issues. That way they don't need to worry about immediately having to fix all the linting issues.

I went back to using npm install instead of npm ci because the package-lock.json file no longer matches packages.json since I added eslint. I'm not sure if I should have uploaded an updated package-lock.json file instead, but it seems to make the pipeline a bit more robust in case someone forgets to update their package-lock.json. Let me know which way would be better.